docs/source/upgrade_to_newest_version.md

```diff
@@ -54,7 +54,7 @@ docker run --rm -v /opt/backup/$PEER_CONTAINER/:/var/hyperledger/production/ \
             hyperledger/fabric-peer:2.0 peer node upgrade-dbs
 ```
 
-If you are using CouchDB as the state database, also drop the CouchDB database.  This can be done by removing the CouchDB /data volume directory.
+In v2.0 and v2.1, if you are using CouchDB as the state database, also drop the CouchDB database. This can be done by removing the CouchDB /data volume directory.
```

Comment:
In Fabric v2.0 and v2.1, if you are using CouchDB as the state database, you also need to drop the CouchDB database. This can be done by removing the CouchDB /data volume directory.